week3/homework/src/actions/markAsDone.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+async function markAsDone(todo, request, response) {
+  try {
+    let todoList = await todo.list().then(todoList => JSON.parse(todoList));
```

Comment:
This variable is not re-assigned, so you could use `const` for declaration.

Reply:
Thanks... done